### ISSUE TITLE
how to sniff zigbee ?

### ISSUE THREAD

**Issue by Vebryn** [slaesh/cc2652-stick#18]
Hi,

A lot of info about sniffing using old stick. Too few info using new cc2652 stick.

Here is what I did,

* Download Uniflash and [packet sniffer 2](https://www.ti.com/tool/PACKET-SNIFFER)
* Start Uniflash
* Get your dongle model and revision 

![1](https://user-images.githubusercontent.com/8328247/133892062-70bc1c6f-f50b-4ef7-a50f-d28e67fceba9.JPG)

* Erase Entire Flash

![2](https://user-images.githubusercontent.com/8328247/133892064-044c582c-b7b9-4268-880c-d46a8e273bbd.JPG)

* Perform blank check

![3](https://user-images.githubusercontent.com/8328247/133892066-cca4974b-0213-4a70-9756-74689ad0d8d4.JPG)

* Select cc2652rlp fw ⚠️ ti fw disable bootloader, you will not able to flash again using bootloader ⚠️

![4](https://user-images.githubusercontent.com/8328247/133892068-32c00096-9de0-4976-a259-2b5d48f5b0dc.JPG)

* Flashing log

![5](https://user-images.githubusercontent.com/8328247/133892069-b8d2bec0-c34d-4bcc-897e-bb53fcee3722.JPG)

Stick blue led is blinking, hopelessly device is not found when I start Packet Sniffer 2

![8](https://user-images.githubusercontent.com/8328247/133892075-d5e6d4e9-3563-4bc9-aae4-bdde7b3fda57.JPG)

Here is my com port configuration :

![6](https://user-images.githubusercontent.com/8328247/133892070-0ec1b1db-aed3-4783-94a4-9754dca2aa37.JPG)
![7](https://user-images.githubusercontent.com/8328247/133892073-e2168bf6-e65a-4a70-9a19-018e9c29c177.JPG)

Does someone succeed in sniffing using cc2652rb stick ?
What should I get into packet sniffer 2 port ? Mine is empty
How to get some debug information ?

Best regards.

References
* https://www.zigbee2mqtt.io/information/flashing_via_uniflash.html

**Comment by wintersteiger** [comment]
I have the same problem; I may have bricked my Sonoff dongle by flashing the wrong firmware (`...\SmartRF Packet Sniffer 2\sniffer_fw\bin\cc2652rblp\sniffer_fw.hex`). The CRC check failed after flashing because it is completely unresponsive. I'll see whether I can connect to it via JTAG instead of the USB/serial converter, but I don't have much hope for that to work.

**Comment by Vebryn** [comment]
Hi @wintersteiger,

Even if you leave bootloader enabled, fw will disable flashing using bootloader. In this case, JTAG flashing will working. You need [CC-DEVPACK-DEBUG](https://www.ti.com/tool/CC-DEVPACK-DEBUG).

**Comment by wintersteiger** [comment]
Yeah, I was able to fix it yesterday, just a bit of a procedure with getting all the wires in the right places. The TI software is a bit of a pain to use and frequently failed to make connection, but after many retries it started working.

**Comment by Vebryn** [comment]
Did you manage to sniff your zigbee network ?

**Comment by wintersteiger** [comment]
No, not with the CC2652P, only with a CC2531. In the meanwhile I also received a CC2652RB, which I believe should work with the sniffer firmware that comes with Smart RF Sniffer 2, but I haven't tried it yet.
